### PR TITLE
Update libiconv package

### PR DESCRIPTION
**Pull request by JL2210**

### CODE REVIEW

File: packages/libiconv.rb [hunk 1]
```diff
@@ -3,32 +3,27 @@
...
   
-  depends_on 'glibc'
-  depends_on 'manpages'
-
   def self.build
-    system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system './configure',
+           '--enable-shared',
+           '--enable-static',
```
Comment (uberhacker) [inline packages/libiconv.rb]: Why do we need static libraries?

Comment (JL2210) [reply packages/libiconv.rb]: Statically linking the core packages so as to reduce dependencies.

Comment (uberhacker) [reply packages/libiconv.rb]: This update is still sitting here since I thought you were going to update ruby.

Comment (JL2210) [reply packages/libiconv.rb]: Yeah, I know. Ruby's having some issues doing this right now, look:
```
~snip~
in `require': incompatible library version
~snip~
```

Comment (uberhacker) [reply packages/libiconv.rb]: And this is the exactly the reason I close these kinds of PRs.  I waste a lot of my time testing PRs that break other packages.

Comment (JL2210) [reply packages/libiconv.rb]: This doesn't break anything. It's just Ruby that's being a pain during build.

Comment (uberhacker) [reply packages/libiconv.rb]: You just contradicted yourself.  This breaks ruby.

Comment (JL2210) [reply packages/libiconv.rb]: No, Ruby breaks Ruby. This is fine.

**Comment by JL2210** [comment]
@uberhacker Could you please merge this?

**Comment by uberhacker** [comment]
Uh...not until we have binaries since this is a core package and you have not shared your branch.

**Comment by JL2210** [comment]
Sorry about that. I usually disable edits from maintainers unless it's needed.

**Pull request merged**